lambdas/webhook.py

```diff
@@ -85,11 +102,60 @@ def handle_post_request(event: dict) -> dict[str, object]:
 
 def valid_signature(event: dict) -> bool:
     secret = os.environ["ALMA_CHALLENGE_SECRET"]
-    message = json.dumps(event["body"])
+    message = event["body"]
     try:
         request_signature = event["headers"]["x-exl-signature"]
     except KeyError:
         return False
     message_hash = hmac.digest(secret.encode(), msg=message.encode(), digest="sha256")
     expected_signature = base64.b64encode(message_hash).decode()
     return request_signature == expected_signature
+
+
+def handle_pod_export_webhook(message_body: dict) -> dict[str, object]:
+    if message_body["job_instance"]["status"]["value"] != "COMPLETED_SUCCESS":
+        logger.warning(
+            "POD export job did not complete successfully, may need investigation. "
+            "Returning 200 success response."
```

Comment:
Non-blocking because I can see the logic of returning a 200, but just noting that it feels weird to return a success response when the server did fail to complete the job successfully.

Reply:
So, there's a good catch here but I think it's in a different return value. From the webhook perspective, this is a success: Alma sent us a valid and expected webhook post. The job failing is an internal Alma issue, not a webhook issue. However, that would also be true of the non-JOB_END webhook, which was sending a 400 response, so I just fixed that. Now the only time we return an error response is (correctly) when Alma sends us an invalid webhook request.